services/table-foreign-keys-analyzer.js

```diff
@@ -8,19 +8,41 @@ function TableForeignKeysAnalyzer(databaseConnection, schema) {
     switch (queryInterface.sequelize.options.dialect) {
       case 'postgres':
         query = `
-          SELECT
-            tc.constraint_name,
-            tc.table_name,
-            kcu.column_name,
-            ccu.table_name AS foreign_table_name,
-            ccu.column_name AS foreign_column_name
-          FROM information_schema.table_constraints AS tc
-          JOIN information_schema.key_column_usage AS kcu
-            ON tc.constraint_name = kcu.constraint_name
-          JOIN information_schema.constraint_column_usage AS ccu
-            ON ccu.constraint_name = tc.constraint_name
-          WHERE constraint_type = 'FOREIGN KEY'
-            AND tc.table_name=:table;`;
+        SELECT 
+          table_constraints.constraint_name,
+          table_constraints.table_name,
+          table_constraints.constraint_type AS column_type,
+          key_column_usage.column_name,
+          constraint_column_usage.table_name AS foreign_table_name,
+          constraint_column_usage.column_name AS foreign_column_name,
+          json_agg(uidx.unique_indexes) filter (where uidx.unique_indexes is not null) AS unique_indexes
```

Comment:
to have cleaner js variables it could be really nice that you alias your variables in camelCase. Don't forget that to do SQL in sql you need to use doubles quotes `AS "columnType"`